compat/beds.lua

```diff
@@ -1,5 +1,10 @@
 local bed_bottoms = {"beds:bed_bottom", "beds:fancy_bed_bottom"}
 
+-- sanity checks
+assert(beds)
+assert(beds.spawn)
+assert(beds.save_spawns)
```

Comment:
Maybe show an error message if assert checks fail.

Reply:
> Maybe show an error message if assert checks fail.

I thought about that but letting it crash here ensures that the code does not execute any invalid stuff later, but you are right: a message would be nice, i think there is a second message parameter for the `assert()` command :thinking:

Reply:
Yes, there is. It's a string value, you can print any much kind of messages.